projects/plugins/jetpack/modules/related-posts/jetpack-related-posts.php

```diff
@@ -1611,10 +1695,8 @@ protected function _generate_related_post_context( $post_id ) {
 
 	/**
 	 * Logs clicks for clickthrough analysis and related result tuning.
-	 *
-	 * @return null
 	 */
-	protected function _log_click( $post_id, $to_post_id, $link_position ) {
+	protected function log_click( $post_id, $to_post_id, $link_position ) {
```

Comment:
Looks like you'll need a phpcs ignore here.

```suggestion
	protected function log_click( $post_id, $to_post_id, $link_position ) { // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
```

Reply:
I was actually wondering what `log_click` did, since it seems to just be an empty function that doesn't do or return anything? I tried MGSing but couldn't find any good info.

Reply:
It is used on WordPress.com (where you'll need to rename methods as well), see the `wpcom-related-posts.php` file there.